backend/pkg/count/service/query_builder.go

```diff
@@ -95,3 +95,32 @@ func countCoOccurrencesQueryBuilder(clusterId string, fromTime time.Time, toTime
 		},
 	}
 }
+
+func incrementNonMatchedClusterIds(coOccurringClusterId string, matchedClusterIds []string) map[string]interface{} {
```

Comment:
This is the main query here doing the heavy lifting. Ignore cluster_ids that HAVE been matched, and search for co_cluster_ids of the cluster where the misses occur.